CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contribution Guidelines
+
+Thank you for your interest in contributing to dwave-examples! Before getting
+started, we encourage you to take a look at this document of guidelines.
+
+## What We're Looking For
+
+While we have a preference for application oriented examples, we welcome all
+examples that demonstrate use of Ocean&#8482; SDK tools.
+
+To see our categorized collection of demos, create a free account on the
+[Leap&#8482; Quantum Cloud Service](https://cloud.dwavesys.com/leap/signup/) and take a look at
+[our library of examples](https://cloud.dwavesys.com/leap/examples/) or
+go through the demo code repositories in [dwave-examples](https://github.com/dwave-examples).
+
+## Steps for Contributing
+
+1. Follow GitHub's instructions on [creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
+   Alternatively, clone this repository (or one of our [existing examples](https://github.com/dwave-examples)) as a
+   starting point from which to implement your example.
```

Comment:
Should there not be a step 2 write your example code?

Reply:
I put a bit of a placeholder for this one but I'll update it in a later PR when I have the details of what steps they should take. I need to first decide on the steps though 😂